client/webserver/site/src/js/charts.js

```diff
@@ -74,12 +74,10 @@ export class DepthChart {
     this.wheeled = () => {
       this.wheelLimiter = setTimeout(() => { this.wheelLimiter = null }, 100)
     }
-    this.wheel = this.wheel_.bind(this)
-    bind(this.canvas, 'wheel', this.wheel)
-    this.resize = this.resize_.bind(this)
-    bind(window, 'resize', this.resize)
-    this.click = this.click_.bind(this)
-    bind(this.canvas, 'click', this.click)
+
+    bind(this.canvas, 'wheel', e => { this.wheel(e) })
+    bind(window, 'resize', e => { this.resize(e) })
+    bind(this.canvas, 'click', e => { this.click(e) })
```

Comment:
Is this effectively the same as doing `bind(this.canvas, 'click', click.bind(this))` to get `this` to be the `DepthChart` inside of `click`?

Reply:
That's right. I'm not sure why I had it the old way.